### PR TITLE
Juthika/intake add new daytime contact

### PR DESCRIPTION
**Pull request by juthikahoque**
## Notion ticket link

[Add Daytime Contacts](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)

## Implementation description
* added a function to create a new daytime contact
* added test cases

## Steps to test
1. switch to branch
2. spin up docker containers
3. run `docker exec -it cas_py_backend /bin/bash -c "pip install -e . && pytest"`

## What should reviewers focus on?
* focus on the implementation!

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/python/app/services/implementations/daytime_contact_service.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+class DaytimeContactService(IDaytimeContactService):
+    def __init__(self, logger):
+        self.logger = logger
+
+    def create_new_daytime_contact(self, contact):
+        try:
+            if not contact:
+                raise Exception("Empty child DTO/None passed to add_new_child function")
+            if not isinstance(contact, CreateDaytimeContactDTO):
+                raise Exception("Child passed is not of CreateDaytimeContactDTO type")
```
Comment (kai-wei-mo) [inline backend/python/app/services/implementations/daytime_contact_service.py:16]: suggestion: change "child" to "contact"

File: backend/python/app/services/interfaces/daytime_contact_service.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    @abstractmethod
+    def create_new_daytime_contact(self, contact):
+        """Adds the specified contact to the daytime_contact table and returns
+            a DaytimeContactDTO of the new entry
+        :param contact: the contact to be added
+        :type contact: CreateDaytimeContactDTO
+        :return: DaytimeContactDTO
+        :rtype: DaytimeContactDTO
+        :raises Exception: if input CreateDaytimeContactDTO is not valid or if there was an error
+                           while insertion
```
Comment (jennchenn) [inline backend/python/app/services/interfaces/daytime_contact_service.py:14]:
```suggestion
                           during insertion
```

File: backend/python/tests/functional/test_daytime_contact_service.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import datetime
+from multiprocessing import dummy
```
Comment (jennchenn) [inline backend/python/tests/functional/test_daytime_contact_service.py:2]: are these being used?

**Review by jennchenn — COMMENTED** [review]
few nits

File: backend/python/app/models/daytime_contact.py [hunk 4]
```diff
@@ -8,9 +8,8 @@ class DaytimeContact(db.Model, BaseMixin):
...
     id = db.Column(db.Integer, primary_key=True, nullable=False)
-    contact_first_name = db.Column(db.String, nullable=False)
-    contact_last_name = db.Column(db.String, nullable=False)
+    name = db.Column(db.String, nullable=False)
     address_id = db.Column(db.Integer, db.ForeignKey("addresses.id"), nullable=False)
     address = db.relationship("Address")
-    phone_number = db.Column(db.String, nullable=False)
-    type = db.Column(daytime_contact_type_enum)
+    contact_information = db.Column(db.String, nullable=False)
+    dismissal_time = db.Column(db.String, nullable=False)
```
Comment (jennchenn) [inline backend/python/app/models/daytime_contact.py:15]: sorry do you mind making `dismissal_time` an optional field 🙏

**Review by jennchenn — APPROVED** [review]
lgtm!!

**Pull request merged**